src/config/dynamic-theme-fixes.config

```diff
@@ -12616,6 +12616,23 @@ INVERT
 
 ================================
 
+weather.com
+
+INVERT
+#svg-symbol-cloud
+#svg-symbol-cloud-junior
+#svg-symbol-drop
+#svg-symbol-lightning-bolt
+#svg-symbol-moon
+#svg-symbol-star
+#svg-symbol-sun
+#svg-symbol-sun-ray
```

Comment:
This could be turned into

```CSS
IGNORE INLINE STYLE
[id^="svg-symbol"] path /* maybe `*` instead of `path` */
```

Reply:
`path` and `*` works same.

Reply:
Then let's just go for `[id^="svg-symbol"] path` :)

Reply:
I used `*` 😬